flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -339,7 +368,11 @@ public void testNotifyCreditAvailable() throws Exception {
 
 			// Trigger notify credits availability via buffer response on the condition of an un-writable channel
 			final BufferResponse bufferResponse3 = createBufferResponse(
-				TestBufferFactory.createBuffer(32), 1, inputChannel1.getInputChannelId(), 1);
+				TestBufferFactory.createBuffer(32),
+				1,
+				inputChannel1.getInputChannelId(),
+				1,
+				new NetworkBufferAllocator(handler));
```

Comment:
ditto: reuse previous `NetworkBufferAllocator`

Reply:
Changed to using the same allocator.